forge/routes/auth/index.js

```diff
@@ -539,6 +539,16 @@ module.exports = fp(async function (app, opts, done) {
         }
     })
 
+    app.get('/token/test/:projectId', {
```

Comment:
All of the token generation routes live in `oauth.js` rather than `index.js`. This should move over there.

We have different types of token - not just project tokens - so the path ought to be scoped so we can space to add similar routes for devices etc in the future. 

I suggest: `/account/check/:ownerType/:ownerId`